app/api/views/briefs.py

```diff
@@ -1244,7 +1244,7 @@ def create_brief_response(brief_id):
 def update_brief_response(brief_id, brief_response_id):
     brief = briefs.get(brief_id)
     if not brief or brief.status != 'live':
-        not_found("Invalid brief id '{}'".format(brief_id))
+        not_found("Invalid opportunity id '{}'".format(brief_id))
```

Comment:
There is still some instances of briefs under the update_brief_response function

Reply:
fixed in updated_brief_response